### DISCUSSION TITLE
Links to plugin docs and source code should be made much more prominent

### DISCUSSION THREAD

**Discussion by GenevieveBuckley** [chanzuckerberg/napari-hub#1266] [Ideas]
#### Description
Links to the plugin webpage and plugin source code should be included prominently in the abstract text section of each plugin page, not only behind buttons hidden away in the top right corner of the screen. It seems almost like a dark pattern designed to keep users only interacting with plugins via the hub page.

![Screenshot 2023-08-14 at 11 40 20 am copy 2](https://github.com/chanzuckerberg/napari-hub/assets/30920819/e2563b19-abcd-499d-a86f-af27168dba2f)

The problems with the current icon-only situation are:
* It's difficult to find this information. It's even more difficult if you are new, and it's possible to miss entirely.
* When you do find it, you have to spend several clicks to open the page you need (a few to see the dropdowns for each icon, and then to click through to the actual plugin webpage)
* The icons are not intuitive. At best, `</>` for "code" is the only one that sort of seems like it's related, but honestly I never see this used elsewhere so I don't remember it here either.
    ![Screenshot 2023-08-14 at 11 46 36 am](https://github.com/chanzuckerberg/napari-hub/assets/30920819/0b6aef33-e601-47b6-b8dc-2ccc715f5134)
* The location of the icons is also not ideal. Looking at the UI, the most important pieces of information go into the biggest, central column. Secondary information has been put into the left hand side bar. I do not expect to find really critical pieces of information (the plugin project page and source code links) tucked away on the right hand side (because this is the third tier of information, I don't recognize the icons, and the right hand side looks mostly blank or has internal links to different subheadings on the same page)

I think it's really important that new users are (1) directed to the full documentation websites for plugins, (2) see the actual source code, and (3) know where to report issues directly to the plugin authors. That's why I'd like to see the hub page improved.

#### Expected Results
I'd expect to see lines like this in the abstract after the one line description and before the plugin author names.

> Plugin website: www.plugin-amazing-docs.com
> Plugin source code: www.github.com/super-awesome/napari-plugin-foo

![Screenshot 2023-08-14 at 11 52 05 am](https://github.com/chanzuckerberg/napari-hub/assets/30920819/f0ffbd3b-1dd6-447b-9a2c-4d39063ebd37)

*Note: I'm not suggesting getting rid of the icons in the corner if you don't want to. This issue is about making sure that critical pieces of information are displayed in a clear and obvious manner (i.e. adding the info to the abstract in the centre of the screen).

**Comment by jni** [comment]
I just want to add my support for this request: actually I myself have run into it and I don't remember even finding the links — I think I just did a web search for "[plugin name] github" to find the project source. That's definitely not the UX we want!

I agree with all of Genevieve's concerns — the icons do seem relatively standard (for example, if I do an image search for "icon website" it's either a globe, a globe with "www" overlaid, or a globe with a cursor overlaid; the latter two of these would be better imho for being more explicit), but not that well known. Neither Genevieve nor I knew them (especially not "source code"), and we're not particularly naive users here...

As an additional idea, the stats bar that shows "Github activity" could directly link to the Github page for the project. e.g. the word "Github" could link to the source code, pull requests could link to the pull requests page, and issues could link to the issues.

Hopefully all of these are quite easy to implement... 🙏

**Comment by dgmccart** [comment]
Thank you for posting [this issue](https://github.com/chanzuckerberg/napari-hub/issues/1217), @GenevieveBuckley, and @jni for adding your thoughts. We appreciate your feedback and concern. I've converted it to a discussion since we'll be compiling ideas and feature requests on the discussion board going forward.

We switched to icon-based navigation following feedback from users to simplify the developer’s description of the plugin, but we’re open to revisions in the future to make it easier for users to quickly get to Github and external documentation. For the near future, however, we’re committed to primarily fixing things that may break the site and supporting existing features. Our designers and engineers are tied up on another project with a tight end-of-year timeline and so have to focus on meeting that deadline before we return to hub enhancements and deeper support. 

If you want to talk more about how others outside of CZI can contribute to the hub, please let me know.
